lib/matplotlib/tight_layout.py

```diff
@@ -262,42 +247,34 @@ def get_subplotspec_list(axes_list, grid_spec=None):
 def get_tight_layout_figure(fig, axes_list, subplotspec_list, renderer,
                             pad=1.08, h_pad=None, w_pad=None, rect=None):
     """
-    Return subplot parameters for tight-layouted-figure with specified
-    padding.
-
-    Parameters:
-
-      *fig* : figure instance
-
-      *axes_list* : a list of axes
-
-      *subplotspec_list* : a list of subplotspec associated with each
-        axes in axes_list
-
-      *renderer* : renderer instance
-
-      *pad* : float
-        padding between the figure edge and the edges of subplots,
-        as a fraction of the font-size.
-
-      *h_pad*, *w_pad* : float
-        padding (height/width) between edges of adjacent subplots.
-        Defaults to `pad_inches`.
-
-      *rect* : if rect is given, it is interpreted as a rectangle
-        (left, bottom, right, top) in the normalized figure
-        coordinate that the whole subplots area (including
-        labels) will fit into. Default is (0, 0, 1, 1).
+    Return subplot parameters for tight-layouted-figure with specified padding.
+
+    Parameters
+    ----------
+    fig : Figure
+    axes_list : List[Axes]
+    subplotspec_list : List[SubplotSpec]
+        The subplotspecs of each axes.
```

Comment:
Can this be "\`.SubplotSpec\`" so there is a link to what a SubplotSpec is?

Reply:
changed back to old form for now